src/ui/components/touchableLink/InternalTouchableLink.tsx

```diff
@@ -33,9 +31,9 @@ export const InternalTouchableLink: FunctionComponent<InternalTouchableLinkProps
   }, [navigate, push, reset, fromRef, withPush, withReset, params, screen])
   return (
     <TouchableLink
-      handleNavigation={enableNavigate && isFocused ? handleNavigation : undefined}
-      onBeforeNavigate={isFocused ? onBeforeNavigate : undefined}
-      onAfterNavigate={isFocused ? onAfterNavigate : undefined}
+      handleNavigation={enableNavigate ? handleNavigation : undefined}
+      onBeforeNavigate={onBeforeNavigate}
+      onAfterNavigate={onAfterNavigate}
```

Comment:
je ne serais pas étonné que ça soit l'une des raisons qui ferait que @arenault-pass a besoin de cliquer 2 fois dans les tests end to end